test/integration/root-optional-revalidate/test/index.test.js

```diff
@@ -26,7 +26,8 @@ const runTests = () => {
     const props = await getProps('/')
     expect(props.params).toEqual({})
 
-    await waitFor(1000)
+    // revalidate and some buffer for the page to be ISR'd
+    await waitFor(1000 + 100)
```

Comment:
For these maybe we add a log in the incremental cache in test mode that signals when the cache has been written so we can retry() looking for that log?